### PR TITLE
Adewyer working

### PR DESCRIPTION
**Pull request by adewyer**
Updated conformers.py to compare theoretical # of conformers to conformer limit (random_conf parameter). If theoretical confs > limit then run with limited number.

Additionally code looks at number of ring conformers and only generates a limited number of conformers per ring. i.e if there are 3 ring conformers and total conformers = 100 then the equation would be:

# conformers/ring = (total conformers/ring conformers) + 2 = (100/3) + 2 = 33 + 2 = 35 conformers generated per ring structure.

### CODE REVIEW

**Comment by adewyer** [comment]
Also output for the kinbot.log file pertaining to changes in the stereochemistry of the inchi label for wells and products has been added as well. If the stereochemistry changes then a warning is printed in kinbot.log.

File: kinbot/conformers.py [hunk 1]
```diff
@@ -207,7 +205,16 @@ def generate_conformers(self, rotor, cart):
...
         rotor: the rotor number in the order it was discovered
         """
-        if len(self.species.conf_dihed) > self.max_dihed:
+        
+        if self.cyc_conf == 0:
+            cycles = 1
+        else:
+            cycles = self.cyc_conf
+       
+        print(len(self.species.conf_dihed), self.cyc_conf)
```
Comment (juditzador) [inline kinbot/conformers.py:214]: here and two lines below a print command should be removed

File: kinbot/conformers.py [hunk 2]
```diff
@@ -247,7 +254,11 @@ def generate_conformers_random_sampling(self, ini_cart):
         """
         Generate a random sampling of each dihedral for a number nconfs of conformers
         """
-        for ni in range(self.nconfs):
+        self.nconfs_new = self.nconfs
+        if self.cyc_conf > 1:
+            self.nconfs_new = int(round(self.nconfs/self.cyc_conf) + 2)
+        print("{} {}".format(self.nconfs, self.nconfs_new))
```
Comment (juditzador) [inline kinbot/conformers.py:260]: print

File: kinbot/homolytic_scissions.py [hunk 3]
```diff
@@ -27,6 +27,7 @@ def create_geometries(self):
         # copy the reactant stationary point
         atom = copy.deepcopy(self.species.atom)
         geom = copy.deepcopy(self.species.geom)
+        print("inside homolytic: name = {}, mult = {}".format(self.species.name, self.species.mult))
```
Comment (juditzador) [inline kinbot/homolytic_scissions.py:30]: print

File: kinbot/homolytic_scissions.py [hunk 4]
```diff
@@ -46,7 +47,7 @@ def __init__(self, species, par, qc):
         self.par = par
         # list of homolytic scission objects
         self.hss = []
-
+        print("HS: name = {}, mult = {}".format(self.species.name, self.species.mult))
```
Comment (juditzador) [inline kinbot/homolytic_scissions.py:50]: print

**Pull request merged**